### ISSUE TITLE
classify

### ISSUE THREAD

**Issue by pcb9382** [ultralytics/yolov5#9114]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

python3 classify/train.py --model yolov5s-cls.pt --data cifar100--epochs 5 --img 224 --batch 64

`
wandb: Currently logged in as: anony-mouse-317294. Use `wandb login --relogin` to force relogin
classify/train: model=yolov5s-cls.pt, data=mnist, epochs=5, batch_size=64, imgsz=224, nosave=False, cache=None, device=, workers=8, project=runs/train-cls, name=exp, exist_ok=False, pretrained=True, optimizer=Adam, lr0=0.001, decay=5e-05, label_smoothing=0.1, cutoff=None, dropout=None, verbose=False, seed=0, local_rank=-1
github: skipping check (not a git repository), for updates see https://github.com/ultralytics/yolov5
Python 3.7.0 required by YOLOv5, but Python 3.6.9 is currently installed
YOLOv5 🚀 2022-8-22 Python-3.6.9 torch-1.9.0+cu111 CUDA:0 (NVIDIA GeForce RTX 3060 Laptop GPU, 5938MiB)

TensorBoard: Start with 'tensorboard --logdir runs/train-cls', view at http://localhost:6006/
Traceback (most recent call last):
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 996, in init
    wi.setup(kwargs)
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 251, in setup
    settings._apply_init(kwargs)
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_settings.py", line 1468, in _apply_init
    self.update(init_settings, source=Source.INIT)
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_settings.py", line 1176, in update
    self.__dict__[key].update(value, source)
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_settings.py", line 331, in update
    self._value = self._validate(self._preprocess(value))
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_settings.py", line 297, in _validate
    f"Invalid value for property {self.name}: {value}"
ValueError: Invalid value for property project: runs/train-cls
wandb: ERROR Abnormal program exit
Traceback (most recent call last):
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 996, in init
    wi.setup(kwargs)
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 251, in setup
    settings._apply_init(kwargs)
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_settings.py", line 1468, in _apply_init
    self.update(init_settings, source=Source.INIT)
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_settings.py", line 1176, in update
    self.__dict__[key].update(value, source)
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_settings.py", line 331, in update
    self._value = self._validate(self._preprocess(value))
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_settings.py", line 297, in _validate
    f"Invalid value for property {self.name}: {value}"
ValueError: Invalid value for property project: runs/train-cls

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "classify/train.py", line 331, in <module>
    main(opt)
  File "classify/train.py", line 317, in main
    train(opt, device)
  File "classify/train.py", line 71, in train
    logger = GenericLogger(opt=opt, console_logger=LOGGER) if RANK in {-1, 0} else None
  File "/home/pcb/yolov5-v6.2/utils/loggers/__init__.py", line 257, in __init__
    config=opt)
  File "/home/pcb/.local/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 1037, in init
    raise Exception("problem") from error_seen
Exception: problem
`

**Comment by AyushExel** [comment]
@pcb9382 you can solve this by passing a project name like this 
```
python3 classify/train.py --model yolov5s-cls.pt --data cifar100--epochs 5 --img 224
 --batch 64 --project yolov5-cls
```
I'll push permanent fix for this

**Comment by glenn-jocher** [comment]
@pcb9382 good news 😃! Your original issue may now be fixed ✅ in PR #9117 by @AyushExel. To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
